Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3662,8 +3662,28 @@ script:
       name: description
     - description: The serial number for a virtual system commit. If provided, the commit will be a virtual system commit.
       name: serial_number
-    description: Pushes rules from PAN-OS to the configured device group. In order to push the configuration to Prisma Access managed tenants (single or multi tenancy), use the device group argument with the device group which is associated with the tenant ID.
+    - auto: PREDEFINED
+      description: Whether to use polling.
+      defaultValue: 'false'
+      name: polling
```

Comment:
calling those arguments `polling` is one of our standards, no? for `pan-os-commit` the polling argument is simply called `polling`.

Reply:
shouldn't they be aligned to each other (`pan-os-commit` and `pan-os-push-to-device-group`)? calling one `verify_push_successful` while for the `pan-os-commit` its called `polling` is a good idea?

Reply:
think youre right since panos should be internally consistent. Im not sure we have a standard regarding the naming of this param
In general I think it makes sense that an arg should describe _what_ it does not _how_ it does it

Reply:
got, i agree with you, keeping it as `polling` ATM